2016-05-31 15:53:09,312 [WARN ][org.jppf.utils.configuration.PropertiesLoader.readInclude(143)]: Could not read [file] 'jppf-common.properties' specified in include statement '#!include file jppf-common.properties', reason: java.lang.NullPointerException: null
2016-05-31 15:53:09,614 [INFO ][org.jppf.utils.ManagementUtils.<clinit>(153)]: management successfully initialized
2016-05-31 15:53:09,672 [INFO ][org.jppf.utils.VersionUtils.logVersionInformation(80)]: --------------------------------------------------------------------------------
2016-05-31 15:53:09,673 [INFO ][org.jppf.utils.VersionUtils.logVersionInformation(81)]: JPPF Version: 5.1.3, Build number: 1762, Build date: 2016-03-24 07:06 CET
2016-05-31 15:53:09,674 [INFO ][org.jppf.utils.VersionUtils.logVersionInformation(82)]: starting client with PID=2128, UUID=168013FD-6D2D-2547-CDC9-A5EF0B1A170D
2016-05-31 15:53:09,675 [INFO ][org.jppf.utils.VersionUtils.logVersionInformation(83)]: --------------------------------------------------------------------------------
2016-05-31 15:53:12,200 [INFO ][org.jppf.client.AbstractGenericClient.newConnection(281)]: connection [jppf_discovery-1-1] created
2016-05-31 15:53:12,252 [INFO ][org.jppf.client.ClassServerDelegateImpl.init(76)]: [client: jppf_discovery-1-1 - ClassServer] Attempting connection to the class server at LAPTOP-4P9M99I4:11111
2016-05-31 15:53:12,308 [INFO ][org.jppf.client.ClassServerDelegateImpl.init(84)]: [client: jppf_discovery-1-1 - ClassServer] Reconnected to the class server
2016-05-31 15:53:12,310 [INFO ][org.jppf.client.TaskServerConnectionHandler.init(78)]: [client: jppf_discovery-1-1 - TasksServer] Attempting connection to the task server at LAPTOP-4P9M99I4:11111
2016-05-31 15:53:12,464 [INFO ][org.jppf.client.TaskServerConnectionHandler.init(91)]: [client: jppf_discovery-1-1 - TasksServer] Reconnected to the JPPF task server
2016-05-31 15:55:10,012 [INFO ][org.jppf.utils.VersionUtils.logVersionInformation(80)]: --------------------------------------------------------------------------------
2016-05-31 15:55:10,014 [INFO ][org.jppf.utils.VersionUtils.logVersionInformation(81)]: JPPF Version: 5.1.3, Build number: 1762, Build date: 2016-03-24 07:06 CET
2016-05-31 15:55:10,015 [INFO ][org.jppf.utils.VersionUtils.logVersionInformation(82)]: starting client with PID=2128, UUID=9ABCB666-33D4-42DA-B79C-DC0BB35ED0D1
2016-05-31 15:55:10,016 [INFO ][org.jppf.utils.VersionUtils.logVersionInformation(83)]: --------------------------------------------------------------------------------
2016-05-31 15:55:10,997 [INFO ][org.jppf.client.AbstractGenericClient.newConnection(281)]: connection [jppf_discovery-1-1] created
2016-05-31 15:55:11,000 [INFO ][org.jppf.client.ClassServerDelegateImpl.init(76)]: [client: jppf_discovery-1-1 - ClassServer] Attempting connection to the class server at LAPTOP-4P9M99I4:11111
2016-05-31 15:55:11,011 [INFO ][org.jppf.client.ClassServerDelegateImpl.init(84)]: [client: jppf_discovery-1-1 - ClassServer] Reconnected to the class server
2016-05-31 15:55:11,012 [INFO ][org.jppf.client.TaskServerConnectionHandler.init(78)]: [client: jppf_discovery-1-1 - TasksServer] Attempting connection to the task server at LAPTOP-4P9M99I4:11111
2016-05-31 15:55:11,026 [INFO ][org.jppf.client.TaskServerConnectionHandler.init(91)]: [client: jppf_discovery-1-1 - TasksServer] Reconnected to the JPPF task server
